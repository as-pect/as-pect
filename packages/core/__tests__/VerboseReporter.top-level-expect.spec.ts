import { createReporterModule } from "./setup/createReporterModule";
import { VerboseReporterWrapper } from "./setup/VerboseReporterWrapper";
const reporter = new VerboseReporterWrapper();

beforeAll(
  () =>
    new Promise<void>((resolve, reject) => {
      createReporterModule(
        "./assembly/jest-reporter-top-level-expect.ts",
        {},
        (err, _result) => {
          if (err) {
            console.log(err);
            reject(err);
          } else {
            resolve();
          }
        },
        reporter,
      );
    }),
);

test("snapshots", () => {
  for (const [name, values] of reporter.snapshots.entries()) {
    for (const value of values) {
      expect(value).toMatchSnapshot(name);
    }
  }
});
