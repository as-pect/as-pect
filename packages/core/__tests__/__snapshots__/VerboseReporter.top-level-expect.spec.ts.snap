// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`snapshots: onFinish assembly/jest-reporter.ts 1`] = `""`;

exports[`snapshots: onGroupEnd  1`] = `""`;

exports[`snapshots: onGroupStart  1`] = `""`;
